templates/layout/page.html.twig

```diff
@@ -38,7 +38,18 @@
   <div class="ecl-container">
     <div class="ecl-row">
       <div class="ecl-col-sm-12">
-        {{ page.content }}
+        {% if page.left_sidebar %}
```

Comment:
Let's not handle the logic regarding columns and new region directly in this PR. In terms of current regions, just guarantee the data attribute is added to the content region, so that h2 elements are taken in consideration.

Reply:
Reverted. Changes in page layout will go to ewcms.